libkbfs/folder_branch_ops.go

```diff
@@ -3142,13 +3149,19 @@ func (fbo *folderBranchOps) reembedForFBM(ctx context.Context,
 
 type applyMDUpdatesFunc func(context.Context, *lockState, []*RootMetadata) error
 
+var errUnmerged = errors.New("fbo is on an unmerged local revision")
```

Comment:
We have a bunch of error types in `errors.go`.  You could add a new type for this, and that way the check would be a simple type check, rather than a string check.  It probably doesn't matter much, but it would be a little more consistent with the rest of our code.